sherpa-onnx/csrc/CMakeLists.txt

```diff
@@ -94,15 +92,20 @@ set(sources
   wave-reader.cc
 )
 
-list(APPEND sources
-  lexicon.cc
-  offline-tts-impl.cc
-  offline-tts-model-config.cc
-  offline-tts-vits-model-config.cc
-  offline-tts-vits-model.cc
-  offline-tts.cc
-  wave-writer.cc
-)
+
+if(SHERPA_ONNX_ENABLE_TTS)
+  list(APPEND sources
```

Comment:
How about to build a separate library for tts, e.g., `libsherpa-onnx-core-tts.so`?

If you only want to use the ASR part, then you don't need to link with `libsherpa-onnx-core-tts.so`.

Reply:
Yes. That makes more sense.